### DISCUSSION TITLE
Week #13 - Final Submission and Reflection

### DISCUSSION THREAD

**Discussion by bst3ck** [USU-ITLS-x265-23-summer/class-repo#239] [General]
## Final Submission

I have really appreciated this course and appreciate being able to participate with all of you in this class. My final project can be viewed here: [http://bksteckler.com/klokke/](http://bksteckler.com/klokke/) with the repo link here: [https://github.com/USU-ITLS-x265-23-summer/bst3ck-final-project](https://github.com/USU-ITLS-x265-23-summer/bst3ck-final-project).

I have reviewed a lot of the sites from the various teams and I have seen a lot of great work. I will not call out anyone specifically, but I know a few people who were struggling at the beginning of the course and to see their projects near the finish line, I am very impressed with their progress. I have seen a lot of growth from the various team members. I know not everyone will continue on the path to becoming a web developer but sticking with it to this point shows a lot. Well done everyone.

### Final Thoughts and Project Self Assessment

I have tried to use a lot of what we learned in this class, but one item that I found really helpful was a javascript library called jquery. I did some research on this and it seems like there is a love-hate relationship with this library. It helps to streamline some of the javascript code, but I guess it adds some weight to the site load. But with all the API calls that were being done on this site, it really sped up the process.

Overall, if I was to start this over again, I think I would have done a lot of things differently. But for an initial concept, I think it worked out. 

Here is what I felt I was able to accomplish
Rubric table  | Basic | Intermediate | Advanced |
|---------------|:-----:|:------------:|:---------:|
| HTML          |      |  x  |          |
| CSS           |      |  x  |          |
| JavaScript    |     |            |  x  |
| Accessibility |       |      x        |         |
| API |       |              |    x     |
| GIT |       |        x      |         |

Thanks again for this class and good luck everyone.

**Comment by audreyhki** [comment]
Great job this semester, I like that you included the rubric and thoughts on how well you accomplished the tasks you were given this semester.
